src/modules/Channel/components/MessageList/index.tsx

```diff
@@ -238,6 +241,17 @@ const MessageList: React.FC<MessageListProps> = ({
                 );
               })
             }
+            {
+              !hasMoreNext
+              && store?.config?.groupChannel?.enableTypingIndicator
+              && store?.config?.groupChannel?.typingIndicatorTypes?.has(TypingIndicatorTypes.Bubble)
+              && <TypingIndicatorMessage
+                typingMembers={typingMembers.filter((typingMember: Member) => (
+                  typingMember.userId !== store?.config?.userId
+                ))}
```

Comment:
Does typing members include current user?

Reply:
Ah no. Thanks for catching this redundancy. i remove it